packages/utils/layoutContext/LayoutContextHelpers.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import { LAYOUT } from './LayoutConstants';
+
+// $FlowFixMe Object.values() returns mixed[] when it should return number[] https://github.com/facebook/flow/issues/2221
+const layoutValues: Array<number> = Object.values(LAYOUT).sort((a, b) => a - b);
+
+export const getLayout = (windowWidth: number): number => {
```

Comment:
I think that it will be better if getLayout will be a pure function. And what about use reduce instead of for each? But it is just an idea. 
something like this:
```
layoutValues.reduce((result, value) => {
  if(windowWidth >= value) {
     return result = value
  }
   return result
}, LAYOUT.largeDesktop  )
```

Reply:
Sure, updated to reduce. Thanks